ui/src/app/modules/teams/components/actions-column/actions-column.component.ts

```diff
@@ -63,38 +66,38 @@ export class ActionsColumnComponent implements OnInit {
       );
     });
 
-    this.actionItemChanged.subscribe((response) => {
-      const actionItem = response.payload as ActionItem;
+    this.actionItemChanged.subscribe((response) =>
+      this.processActionItemChange(response)
+    );
+  }
 
+  processActionItemChange(response: WebsocketResponse) {
+    function retrieveActionItemFromPayload(message: WebsocketResponse) {
```

Comment:
I have read a lot of articles suggesting that you **should** scope a function in another function if that is the only usage.  Ultimately, I think it's a matter of preference.

I updated the inner function to refer to the `message` object instead of the `response` object